src/components/MessageList/TypingIndicatorContainer.js

```diff
@@ -0,0 +1,32 @@
+import React, { useContext } from 'react';
+import styled from '@stream-io/styled-components';
+
+import { ChannelContext, ChatContext } from '../../context';
+
+const Container = styled.View`
+  position: absolute;
+  bottom: 0;
+  height: 30px;
+  width: 100%;
+  padding-left: 16px;
+  padding-top: 3px;
+  padding-bottom: 3px;
+  ${({ theme }) => theme.messageList.typingIndicatorContainer.css}
+`;
+
+const TypingIndicatorContainer = ({ children }) => {
+  const { client } = useContext(ChatContext);
+  const { typing } = useContext(ChannelContext);
+  const typingUsers = Object.values(typing);
+
+  if (
+    typingUsers.length === 0 ||
```

Comment:
doing this `typingUsers.length === 0` instead of this `!typingUsers.length` for a reason? I guess it's more verbose but just wondering